testdata/p4_16_errors_outputs/enumCast.p4-stderr

```diff
@@ -4,13 +4,13 @@ enumCast.p4(26): [--Werror=type-error] error: y: values of type 'int' cannot be
 enumCast.p4(3)
 enum bit<32> X {
              ^
-enumCast.p4(27): [--Werror=type-error] error: AssignmentStatement: values of type 'bit<32>' cannot be implicitly cast to type 'X'
+enumCast.p4(27): [--Werror=type-error] error: y = 32w1: values of type 'bit<32>' cannot be implicitly cast to type 'X'
         y = 32w1; // Error: no implicit cast to enum
           ^
 enumCast.p4(3)
 enum bit<32> X {
              ^
-enumCast.p4(33): [--Werror=type-error] error: AssignmentStatement: values of type 'E2' cannot be implicitly cast to type 'E1'
+enumCast.p4(33): [--Werror=type-error] error: a = b: values of type 'E2' cannot be implicitly cast to type 'E1'
```

Comment:
It looks like the assignment statement here is missing quotes, which makes it harder to read.

Reply:
Do you mean like `"a = b"`?

Reply:
Yes

Reply:
Are you requesting to add quotes only to the error messages validated in the tests that this PR affects, or to _all_ error messages that might possibly print an `IR::AssignmentStatement`?

Reply:
It is probably prudent to only add quotes to the error messages that are changed in this PR for now. Changing all error messages might be too much too ask for.